docs/products/opensearch/howto/sample-dataset.rst

```diff
@@ -89,54 +90,68 @@ OpenSearch Python client offers a helper called bulk() which allows us to send m
 
     python epicurious_recipes_import.py
 
-Sample queries with HTTP client
--------------------------------
 
-With the data in place, we can start trying some queries against your OpenSearch service. Since it has a simple HTTP interface, you can use your favorite HTTP client. In these examples, we will use `httpie <https://github.com/httpie/httpie>`_ because it's one of our favorites.
+.. _get-mapping-with-python:
 
-First, export the ``SERVICE_URI`` variable with your OpenSearch service URI address and index name from the previous script:
+Get data mapping with Python
+----------------------------
+When we sent the data to OpenSearch cluster on :ref:`load the data with Python <load-data-with-python>`, no data structure was specified.
+This is possible because OpenSearch uses **dynamic mapping** to automatically add the fields to their type mapping. 
```

Comment:
We opted here to use dynamic mapping. I feel that it makes sense to be a bit more specific that we let OpenSearch to use dymanic mapping